product_docs/docs/biganimal/release/using_cluster/02_connecting_your_cluster/connecting_from_gcp/index.mdx

```diff
@@ -57,76 +57,75 @@ To walk through an example in your own environment, you need a:
 - Subnet in the VM’s VPC in the same region as the BigAnimal cluster.
 
 
-### Step 1: Publish a service from BigAnimal
+#### Step 1: Publish a service from BigAnimal
 
 !!! Note
-    Perform this procedure in the Google Cloud project connected to your BigAnimal subscription.
+    Publish a service from BigAnimal in the Google Cloud project connected to your BigAnimal subscription.
 
-In the Google Cloud project connected to BigAnimal, create a PSC Published Service to provide access to your cluster from other VPCs in other Google Cloud projects. Perform this procedure for each Postgres cluster to which you want to provide access.
+In the Google Cloud project connected to BigAnimal, to provide access to your cluster from other VPCs in other Google Cloud projects, create a PSC published service. Publish a service from BigAnimal for each Postgres cluster to which you want to provide access.
```

Comment:
Published Service is a thing that is capitalized in the Google Cloud docs. Are we sure we don't want to capitalize it in ours? Do you mind checking other things that you made lowercase against the Google Cloud docs and making sure we are doing the right thing by going with lowercase?